### PR TITLE
fix: error black screen when request permission android

### PR DESCRIPTION
**Pull request by renanzdm**

### CODE REVIEW

**Comment by renanzdm** [comment]
@juliansteenbakker

File: android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt [hunk 1]
```diff
@@ -86,16 +87,10 @@ class MobileScanner(
     /**
      * Request camera permissions.
      */
-    fun requestPermission(permissionCallback: PermissionCallback) {
+    fun requestPermission() {
         listener
             ?: PluginRegistry.RequestPermissionsResultListener { requestCode, _, grantResults ->
```
Comment (juliansteenbakker) [inline android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt:92]: The ?: should be an =. I've changed this on the master branch and now everything is working.

**Comment by juliansteenbakker** [comment]
I've fixed this on the master branch so this PR is no longer needed. Thank you however for looking into this and raising this issue.

**Comment by renanzdm** [comment]
I thank you for the package, could you adjust this in  3.0.0-beta.2?

File: android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt (hunk 1 above)
Comment (renanzdm) [reply android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt:92]: I thank you for the package, could you adjust this in 3.0.0-beta.2?

**Pull request closed without merging**